docs/adding_compute_funs.rst

```diff
@@ -114,3 +116,34 @@ as long as it is registered.
 dependencies specified by the decorator. The function itself should do any calculation
 needed using these dependencies and then add the output to the ``data`` dictionary and
 return it. The key in the ``data`` dictionary should match the ``name`` of the quantity.
+
+Once a new quantity is added to the ``desc.compute`` module, there is a final two steps involving the testing suite which must be checked.
+The first step is implementing the correct axis limit, or marking it as not finite or not implemented.
+We can check whether the axis limit currently evalutates as finite by computing the quantity on a grid with a node at the axis.
+::
+
+    from desc.examples import get
+    from desc.grid import LinearGrid
+    import numpy as np
+
+    eq = get("HELIOTRON")
+    grid = LinearGrid(L=1, M=1, N=1, axis=True)
```

Comment:
Would it be useful to include more than a single collocation point on the axis? This grid also contains two flux surfaces: rho=0 and rho=1. I suggest changing to:
```
grid = LinearGrid(M=4, N=8, rho=np.array([0]))
```